migration/migration.go

```diff
@@ -576,43 +580,92 @@ func createOrUpdateWorkItemLinkType(ctx context.Context, linkCatRepo *link.GormW
 	return nil
 }
 
+// createCategory creates category in db
+func createOrUpdateSingleCategory(ctx context.Context, categoryRepo category.Repository, categoryID *uuid.UUID, categoryName string) error {
+	category := category.Category{
+		ID:   *categoryID,
+		Name: categoryName,
+	}
+	_, err := categoryRepo.Create(ctx, &category)
+	if err != nil {
+		return errs.WithStack(err)
+	}
+	return nil
+}
+
+func createOrUpdateCategories(ctx context.Context, db *gorm.DB, categoryRepo category.Repository, categoryID *uuid.UUID, categoryName string) error {
+	cat, err := categoryRepo.LoadCategoryFromDB(ctx, *categoryID)
+	cause := errs.Cause(err)
+	switch cause.(type) {
+	case errors.NotFoundError:
+		err = createOrUpdateSingleCategory(ctx, categoryRepo, categoryID, categoryName)
+		if err != nil {
+			return errs.WithStack(err)
+		}
+	case nil:
+		log.Info(ctx, map[string]interface{}{
+			"category": categoryName,
+		}, "Category %s exists, will update/overwrite all fields", categoryName)
```

Comment:
Correct. I fixed it here -> d0115c1

But even in the case of workitemtypes not all the fields of workitemtypes are updated -> https://github.com/almighty/almighty-core/blob/master/migration/migration.go#L719 

Do you suggest i raise another pr to fix it in the same way as i have done for categories?